Tools/setup/ubuntu.sh

```diff
@@ -216,5 +216,5 @@ fi
 
 if [[ $INSTALL_NUTTX == "true" ]]; then
 	echo
-	echo "Reboot computer before attempting to build NUTTX targets"
+	echo "Reboot computer before attempting to build NuttX targets"
```

Comment:
```suggestion
	echo "Logout and login before attempting to build NuttX targets, or do `source ~/.profile`"
```
```

Reply:
Sure, they should say the same, I was thinking about creating a helper shell script to have no duplicates.

Reply:
Fixed in https://github.com/PX4/Firmware/pull/13111/commits/760a204f7523395b5677dd66cb7bdba99d0045e1